src/AddressRegistry.Api.BackOffice.Abstractions/Validation/Readdress.cs

```diff
@@ -1,9 +1,36 @@
-﻿namespace AddressRegistry.Api.BackOffice.Abstractions.Validation
+namespace AddressRegistry.Api.BackOffice.Abstractions.Validation
 {
+    using StreetName;
+    using TicketingService.Abstractions;
+
     public static partial class ValidationErrors
     {
         public static class Readdress
         {
+            public static class AddressInvalidStatus
+            {
+                public const string Code = "AdresAfgekeurdGehistoreerd";
+                public static string Message(AddressPersistentLocalId addressId) => $"Dit adres '{addressId}' heeft niet status 'voorgesteld' of 'inGebruik'.";
```

Comment:
This has to change to: `Deze actie is enkel toegestaan op adressen met status 'voorgesteld' of 'inGebruik': {PuriOfAddress}.`